tsconfig.client.json

```diff
@@ -0,0 +1,16 @@
+{
+	"extends": "./tsconfig.json",
```

Comment:
Can we just merge whatever's in tsconfig.json into the client/server ones and get rid of tsconfig.json? Or is it worth having that common one? It seems like it'll be hard to know when to put stuff in the common one or not so I'd lean towards getting rid of it.